kuzzleio/kuzzle-admin-console
# Filters css refactor

~Depends on #428~

## What does this PR do ?
This PR aligns the styles of the components touched in #428 with the guidelines specified in `CONTRIBUTING.md`. Specifically, 

* styles have been moved from global `.scss` files to individual component's scoped `style` tag
* CSS classes have been harmonized with the BEM convention
* many useless style rules have been wiped away

### How should this be manually tested?

You should just browse data and use the filters and ensure everything looks fine.
